lib/hackerone/client/report.rb

```diff
@@ -126,6 +147,87 @@ def suggest_bounty(message:, amount:, bonus_amount: nil)
         Activities.build(response_body)
       end
 
+      ## Idempotent: change the state of a report. See STATES for valid values.
+      #
+      # id: the ID of the report
+      # state: the state in which the report is to be put in
+      #
+      # returns an HackerOne::Client::Report object or raises an error if
```

Comment:
I think this returns a hash (but should return an `Report`)

Reply:
You're right, I broke the docstring.